lib/tasks/check_for_bad_time_handling.rake

```diff
@@ -1,25 +0,0 @@
-task check_for_bad_time_handling: :environment do
```

Comment:
It might be helpful to add https://www.rubydoc.info/gems/rubocop/0.57.2/RuboCop/Cop/Rails/TimeZone to the commit message.

Reply:
Sure, I've added that 👍